components/com_kunena/admin/install/plugins/plg_kunena_community/activity.php

```diff
@@ -91,7 +91,7 @@ public function onAfterReply(KunenaForumMessage $message) {
 		$act->cmd = 'wall.write';
 		$act->actor = $message->userid;
 		$act->target = 0; // no target
-		$act->title = JText::sprintf('PLG_KUNENA_COMMUNITY_ACTIVITY_REPLY_WALL',$params->get('actor_url'),$params->get('actor'),$params->get('title_url'),$params->get('title'));
```

Comment:
I believe that this change came from JomSocial team updating our code to their new standards.